app/components/RegisterPage/RegisterForm.jsx

```diff
@@ -0,0 +1,132 @@
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log('Form data:', formData);
+  };
+
+  return (
+    <Container maxWidth="sm">
+      <Typography 
+        variant="h4" 
+        color="#1976D2" 
+        align="center" 
+        style={{ fontWeight: 'bold', marginBottom: '10px', marginTop: '20px' }}>
+        Welcome to CoursePRN
+      </Typography>
+      <Typography variant="subtitle1" align="center" style={{ marginBottom: '20px' }}>
+        Sign up to receive offers and manage your courses better
+      </Typography>
+      <form onSubmit={handleSubmit}>
+        <TextField
+          label="Username"
+          variant="outlined"
+          name="username"
+          fullWidth
+          margin="normal"
+          onChange={handleInputChange}
+          value={formData.username}
+        />
+        <TextField
+          label="Email"
+          variant="outlined"
+          name="email"
+          fullWidth
+          margin="normal"
+          onChange={handleInputChange}
+          value={formData.email}
+        />
+        <FormControl fullWidth variant="outlined" margin="normal">
+          <InputLabel htmlFor="password">Password</InputLabel>
+          <OutlinedInput
+            id="password"
+            type={showPassword ? 'text' : 'password'}
+            name="password"
+            onChange={handleInputChange}
+            value={formData.password}
+            endAdornment={
+              <InputAdornment position="end">
+                <IconButton onClick={handleShowPassword} edge="end">
+                  {showPassword ? <Visibility /> : <VisibilityOff />}
+                </IconButton>
+              </InputAdornment>
+            }
+            label="Password"
+          />
+        </FormControl>
+        <FormControl fullWidth variant="outlined" margin="normal">
+          <InputLabel htmlFor="confirmPassword">Confirm Password</InputLabel>
+          <OutlinedInput
+            id="confirmPassword"
+            type={showConfirmPassword ? 'text' : 'password'}
+            name="confirmPassword"
+            onChange={handleInputChange}
+            value={formData.confirmPassword}
+            endAdornment={
+              <InputAdornment position="end">
+                <IconButton onClick={handleShowConfirmPassword} edge="end">
+                  {showConfirmPassword ? <Visibility /> : <VisibilityOff />}
+                </IconButton>
+              </InputAdornment>
+            }
+            label="Confirm Password"
+          />
+        </FormControl>
+        <Button
+          type="submit"
+          variant="contained"
+          color="primary"
+          fullWidth
+          size="large"
+        >
+          Create Account
+        </Button>
+      </form>
+      <Typography variant="body2" align="center" style={{ marginTop: '20px' }}>
+        Already have an account?{' '}
+        <Link href="/login" color="primary">
```

Comment:
I suggest you should use the Link from nextjs

Reply:
sorry, I don't understand what you mean. I'm using <Link> in Next.js
What wrong?

Reply:
No, you are not. I see that you are using Link from Mui